doc/user_manual/sampler.tex

```diff
@@ -105,6 +105,14 @@ \section{Samplers}
   \end{itemize}
 }
 
+\newcommand{\constantVariablesDescription}
+{
+  \xmlNode{constant}, \xmlDesc{XML node,
+  optional parameter} the user is able to input variable that needs to be kept constant.    
+  For doing this, as many \xmlNode{constant} nodes as needed can be inputted, where the body of the node containing the constant value
```

Comment:
I changed it, but ... you can use "input" or "inputted" (http://dictionary.cambridge.org/us/dictionary/english/input) :D